db.json

```diff
@@ -2203,7 +2256,8 @@
     "extensions": ["azw"]
   },
   "application/vnd.amazon.mobi8-ebook": {
-    "source": "iana"
+    "source": "iana",
+    "extensions": ["azw3","kfx"]
   },
```

Comment:
.azw3 mentioned on https://www.iana.org/assignments/media-types/application/vnd.amazon.mobi8-ebook , but not .kfx